.php_cs

```diff
@@ -70,7 +70,7 @@ return PhpCsFixer\Config::create()
         'single_quote' => true,
         'phpdoc_to_comment' => false,
         'ordered_class_elements' => ['sortAlgorithm'=>'none'],
-        'class_attributes_separation' => ['elements'=>['method']],
+        'class_attributes_separation' => ['elements'=>['property']],
```

Comment:
This line could be removed. Then we fallback on symfony standard which is the same as: 

```
'class_attributes_separation' => ['elements'=>['property', 'method', 'const']],
```

Reply:
Hmm..  when I removed it, nothing changed